src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc

```diff
@@ -13,23 +13,128 @@ summary:
+* **Browser based IDEs:** Bundled browser-based IDEs with language tooling, debuggers, terminal, VCS integration and much more.
+* **Extensible platform:** Bring your own IDE. Define, configure and extend the tools that you need for your application by using Plug-ins. Compatible with VSCode extensions.  
+* **Enterprise Integration:** Multi-user capabilities, includes Keycloak for authentication and integration with LDAP/AD. 
+
+
+=== Workspace Model
+
+Che defines the workspace to be the project code files and all of their dependencies necessary to edit, build, run, and debug them. In our world, we treat the IDE and the development runtime as a dependency of the workspace. These items are embedded and included with the workspace, whereever it may run. This compares to classic workspace definitions which may include the project code, but require the developer to bind their IDE and use their laptop localhost to provide a runtime.
+
+Each workspace is isolated from one another and is responsible for managing the lifecycle of the components that are contained within it.
+
+Developers using Eclipse Che use their containers directly in their developer workspaces. **Che workspaces are kubernetes pods which allow to replicate the application runtimes (and its microservices) used in production** and provide a “dev mode” layer on top of those, adding intellisense and IDE toolings.
+
+
+=== Browser based IDEs
+
+Eclipse Che provides browser based IDEs for its workspaces. The default browser based IDE is built on link:https://github.com/theia-ide/theia[Eclipse Theia], with the following capabilities:
+
+* **Monaco based editor:** Fast and responsive editor, codelens and much more.
+* **Language Server Protocol:** Get languague tooling.
+* **Debug Adapter Protocol:** Get debugger tooling.
+* **Command Palette:** Do everything from your keyboard.
+* **Task Support:** Tasks from VS Code are extended and support Che Commands.
+* **Embedded Preview:** Preview your application directly from the IDE, including Markdown preview.
+* **Customizable layout:** Adapt the layout using drag and drop.
+* **And much more:** Outline View, Search, Git.
+
+
+**Different IDEs for different use cases**
+
+
+In Eclipse Che, the IDE is completely decoupled so that it is possible to plug a different IDE into a Che workspaces:
+
+* It can be based on Eclipse Theia (as it is a framework to build a web IDE), such as the popular Sirius on the web:
+
+video::B6aCqywKpyY[youtube]
+
+* Or it can be a completely different solutions, like Jupyter or Eclipse Dirigible
+
+video::VooNzKxRFgw[youtube]
+
+There are a different situations where the default IDE will not cover the use cases of your audience, or you might have stakeholders who are using a dedicated tool that covers their needs instead of using an IDE. In the traditional Eclipse IDE world, that was done with RCP applications.
```

Comment:
```suggestion
There are a different situations where the default IDE does not cover the use cases of your audience, or you might have stakeholders who use a dedicated tool that covers their needs instead of an IDE. In the traditional Eclipse IDE world, that was done with Rich Client Platform (RCP) applications.
```